package.json

```diff
@@ -31,7 +31,7 @@
     "jellyfin-noto": "https://github.com/jellyfin/jellyfin-noto",
     "jquery": "^3.4.1",
     "jstree": "^3.3.7",
-    "libass-wasm": "https://github.com/jellyfin/JavascriptSubtitlesOctopus",
+    "libass-wasm": "https://github.com/jellyfin/JavascriptSubtitlesOctopus#4.0.0-jf",
```

Comment:
I wonder whether this is needed given I've tagged https://github.com/jellyfin/jellyfin-web/pull/1005 to be backported as well, won't it create conflicts?